pramen/core/src/main/scala/za/co/absa/pramen/core/utils/JdbcNativeUtils.scala

```diff
@@ -127,7 +127,16 @@ object JdbcNativeUtils {
     val connection = getJdbcConnection(jdbcConfig, url)
 
     val statement = try {
-      connection.createStatement(ResultSet.TYPE_SCROLL_INSENSITIVE, ResultSet.CONCUR_READ_ONLY)
+      connection.setAutoCommit(jdbcConfig.autoCommit)
```

Comment:
This is the first fix for PostgreSQL. It should be `false`. Made it default, since we never need to commit when ingesting. But it is configurable just in case.